### PR TITLE
Import changes from LexiFi

### PR DESCRIPTION
**Pull request by AurelienSaue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ Installation (with OPAM)
 ------------------------
 
 ```
-opam install ocaml-vdom
+opam install vdom
```
Comment (alainfrisch) [inline README.md:35]: The OPAM package seems to be ocaml-vdom, no? 

Cf https://opam.ocaml.org/packages/ocaml-vdom/

Comment (AurelienSaue) [reply README.md:35]: The package was renamed internally by @nojb. 
Would you rather not rename the public one?

Comment (nojb) [reply README.md:35]: About this point:

- The convention is indeed not to call packages `ocaml-XXX` but `XXX`
- If I am not wrong, Dune enforces that the package name and the library name coincide.

If we care about backwards-compatibility, one possibility is to try defining a dummy package `ocaml-vdom` which depends on the `vdom` package. But am not sure it is worth the complexity...

Comment (alainfrisch) [reply README.md:35]: On https://opam.ocaml.org/packages/, we can see many packages starting with `ocaml-` (and also with just `ocaml`).  I understand this is not recommended, but is the renaming worth the trouble?  In particular, it's good that users will be encouraged by OPAM to upgrade to the new version if they had the previous version installed, no?

Comment (nojb) [reply README.md:35]: From a quick look, there seems to be a convention to deal with this renaming: define an `ocaml-vdom.transition` package (`transition` is the "version"), that depends on `vdom` and shows a nice message explaining the renaming. See for example:

- https://opam.ocaml.org/packages/ocaml-xdg-basedir/
- https://opam.ocaml.org/packages/ocaml-markdown/

Comment (alainfrisch) [reply README.md:35]: Aren't the library names those defined explicitly in lib/dune?

Comment (nojb) [reply README.md:35]: Yes, but Dune complains if the _public_ library names (= Findlib names) in package `foo` are not named `foo` or `foo.XXX`.

Anyway, I think @AurelienSaue is leaning towards following the official instructions for renaming a package: https://opam.ocaml.org/doc/Tricks.html#How-to-rename-a-package.

**Pull request merged**